v3/phentech/rpk_001/rpk_001.json

```diff
@@ -0,0 +1,279 @@
+{
+    "name": "RPK-001",
+    "vendorId": "0xAC12",
+    "productId": "0x2236",
+    "keycodes": ["qmk_lighting"],
+    "menus": [

```

Comment:
your keyboard in QMK firmware has no RGB animations enabled. Please enable them.

Reply:
ok

Reply:
https://github.com/qmk/qmk_firmware/pull/22120 @Xelus22

Reply:
Please simplify to just `"menus": [ "qmk_rgb_matrix" ]`, since you have ALL effects enabled